docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -52,6 +52,7 @@ class UpdateChecker < Dependabot::UpdateCheckers::Base
           #{VERSION_WITH_SFX}|
           #{VERSION_WITH_PFX_AND_SFX}
       /x
+      DIGEST = /^sha256:[0-9a-f]{64}$/
```

Comment:
Calling this the digest makes sense, and we use the same term in the FileParser, but they have different definitions
https://github.com/dependabot/dependabot-core/blob/1cd6b45919e19c6b2aa719015629804302724e18/docker/lib/dependabot/docker/file_parser.rb#L27

Can we try to make the regex consistent between the two?

Reply:
@Nishnha I took my a long time to loop back to this, but I addressed your comment and this PR is now ready for another review!